common/src/java/com/zimbra/common/gql/GqlConstants.java

```diff
@@ -303,4 +303,103 @@
     public static final String LAST_ACCESSED = "lastAccessed";
     public static final String USER_AGENT = "userAgent";
     public static final String REQUEST_IP_ADDRESS = "requestIPAddress";
+
+    // get info constants
+    public static final String GET_INFO_REQUEST = "GetInfoRequest";
```

Comment:
Prefix all of the class names you added here with `CLASS_`. for consistency with the rest. e.g. `CLASS_GET_INFO_REQUEST`, or `CLASS_ACCOUNT_ZIMLET_INFO`, etc (15~ occurrences)